packages/cli/package/src/commands/provider/deploy.ts

```diff
@@ -41,35 +38,7 @@ export default class Deploy extends BaseCommand<typeof Deploy> {
     const { flags } = await initCli(this, await this.parse(Deploy));
     const computePeers = await resolveComputePeersByNames(flags);
 
-    const [invalidPeers, peers] = splitErrorsAndResults(
-      computePeers,
-      ({ kubeconfigPath, ipSupplies, name, ...restPeer }) => {
-        return kubeconfigPath === undefined || ipSupplies.length === 0
```

Comment:
We don't need that?

Reply:
not anymore cause now all of this is required in the config instead of being optional